go/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
```

Comment:
It seems the author no longer maintains this repo and a community maintained version is available here:
https://github.com/dgrijalva/jwt-go 

See more details here:
https://github.com/dgrijalva/jwt-go/issues/462

I think go module is also supported in the community repo.